app/assets/stylesheets/foundation_and_overrides.scss

```diff
@@ -114,9 +114,11 @@ $include-html-global-classes: $include-html-classes;
 // $oil         : #333333;
 // $jet         : #222222;
 // $black       : #000000;
+ $grey          : #7E808F;
+ $lime          : #DDE587;
```

Comment:
Color `#DDE587` should be written as `#dde587`<br>Line should be indented 0 spaces, but was indented 1 spaces